lab-andrew/lib/request-parser.js

```diff
@@ -35,9 +35,6 @@ requestParser.parse = (request) => {
 
     request.on('end',() => {
       try{
-        // vinicio - this is mutating the request object, and creating an
```

Comment:
this shouldn't be in your code at all, the demo code that Vinicio types in class isn't starter-code, its demo code. you should be typing all of this from scratch, not copy / pasting his code.